surfactant/infoextractors/js_file.py

```diff
@@ -126,20 +152,10 @@ def short_name():
     return "js_file"
 
 
-def load_db():
-    js_lib_file = (
-        ConfigManager().get_data_dir_path() / "infoextractors" / "js_library_patterns.json"
-    )
-
-    try:
-        with open(js_lib_file, "r") as regex:
-            database = json.load(regex)
-    except FileNotFoundError:
-        logger.warning(
-            "Javascript library pattern database database could not be loaded. Run `surfactant plugin update-db js_file` to fetch the pattern database."
-        )
-        return None
-    return database
-
-
-js_lib_database = load_db()
+@surfactant.plugin.hookimpl
+def init_hook(command_name=None):
+    """Initialization hook to load the JavaScript library database."""
+    if command_name != "update-db":  # Do not load the database if only updating the database.
+        click.echo("Initializing js_file...")
+        js_db_manager.load_db()
+        click.echo("Initializing js_file complete.")
```

Comment:
The echoes could be logger messages instead -- maybe set to INFO, or DEBUG (to reduce output except when debugging issues)?